app/src/main/java/com/example/marvelapp/data/mapper/CharacterRealmMapper.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.marvelapp.data.mapper
+
+import com.example.marvelapp.data.local.model.CharacterRealmEntity
+import com.example.marvelapp.data.local.model.ComicsRealmEntity
+import com.example.marvelapp.data.local.model.EventsRealmEntity
+import com.example.marvelapp.data.local.model.SeriesRealmEntity
+import com.example.marvelapp.data.local.model.StoriesRealmEntity
+import com.example.marvelapp.data.local.model.ThumbnailRealmEntity
+import com.example.marvelapp.data.local.model.UrlRealmEntity
+import com.example.marvelapp.domain.entity.ComicsEntity
+import com.example.marvelapp.domain.entity.EventsEntity
+import com.example.marvelapp.domain.entity.SeriesEntity
+import com.example.marvelapp.domain.entity.StoriesEntity
+import com.example.marvelapp.domain.entity.ThumbnailEntity
+import com.example.marvelapp.domain.entity.UrlsEntity
+import io.realm.RealmList
+
+object CharacterRealmMapper {
+
+    private fun transformDBCharacter(characterEntity: com.example.marvelapp.domain.entity.CharacterEntity): CharacterRealmEntity =
```

Comment:
Here if you put alt + enter, AS said you to put that com.example.marvelapp..etc in imports

Reply:
Check on other places pls

Reply:
Thx, now its fixed.

![image](https://user-images.githubusercontent.com/73836200/111635565-60b29700-87d6-11eb-86a8-06a087d5bb9c.png)